doc/api/http.md

```diff
@@ -3567,6 +3567,59 @@ server.on('request', (request, res) => {
 server.listen(8000);
 ```
 
+## `http.createStaticServer([options])`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental. The implementation lacks support for partial
+> responses (Ranges) and conditional-GET negotiation (If-Match,
+> If-Unmodified-Since, If-None-Match, If-Modified-Since).
+
+* `options` {Object}
+  * `directory` {string|URL} Root directory from which files would be served.
+    **Default:** `process.cwd()`.
+  * `port` {number}
+  * `host` {string} **Default:** `'localhost'`
+  * `mimeOverrides` {Object} Dictionary linking file extension to a MIME string,
+    to override or extend the built-in ones.
+  * `filter` {Function|null} should be a function that accepts two arguments and
+    returns a value that is coercible to a {boolean} value. When `null`, no
+    files are filtered. **Default:** filters all dot files.
+  * `log` {Function|null} called when sending a response to the client.
+    **Default:** `console.log`.
+  * `onStart` {Function} called the server starts listening to requests.
+    **Default:** logs the URL to the console.
+
+* Returns: {http.Server}
+
+Start a TCP server listening for connections on the given `port` and `host`, and
+serve static local files, using `directory` as the root.
+When specifying a `host` other than `localhost`, you are exposing your local
+file system to all the machines that can connect to your computer.
+
+If specified and not `null`, `filter` will be called with two arguments: the
+first one if the request URL string (the URL that is present in the actual HTTP
+request), and the second one is the `file:` `URL` that was generated from the
+base directory and the request URL. If the function returns a falsy value, the
+request will be blocked.
```

Comment:
I think we should document how it will be blocked

Reply:
```suggestion
base directory and the request URL. If the function returns a falsy value, the
server will respond with a 403 HTTP error.
```